### ISSUE TITLE
GDS GUI is blank if there is no cmdDisp.CMD_NO_OP in the CMD dictionary

### ISSUE THREAD

**Issue by saba-ja** [nasa/fprime#474]
| | |
|:---|:---|
|**_F´ Version_**| 1.5 |
|**_Affected Component_**|   GDS |
---
## Problem Description
GDS GUI is blank if there is no cmdDisp.CMD_NO_OP in the CMD dictionary.
Vue complains for not finding `full_name` of undefined as shown in the chrome devTool below:

![image](https://user-images.githubusercontent.com/35859004/114261795-82480e00-9991-11eb-81f2-ddf51f2a498e.png)

## How to Reproduce

1. Use a dictionary without having `cmdDisp.CMD_NO_OP`

## Expected Behavior

GDS GUI should not break if user does not specify `cmdDisp.CMD_NO_OP`.

**Comment by saba-ja** [comment]
## Root cause
The issue is on line 70 of the following:

https://github.com/nasa/fprime/blob/98f50f21cef5dfa1787e0c22dcf2193f7672c751/Gds/src/fprime_gds/flask/static/js/vue-support/command.js#L68-L77

The default selected command is hardcoded as `cmdDisp.CMD_NO_OP`. If this command does not exist in user's dictionary the value of `selected` will be `undefined` which causes problem on lines 170 and 181 of the following:

https://github.com/nasa/fprime/blob/98f50f21cef5dfa1787e0c22dcf2193f7672c751/Gds/src/fprime_gds/flask/static/index.html#L167-L183

## Correction

the following lines in [owls-bus-fprime/fprime/Gds/src/fprime_gds/flask/static/js/vue-support/command.js](https://github.com/nasa/fprime/blob/98f50f21cef5dfa1787e0c22dcf2193f7672c751/Gds/src/fprime_gds/flask/static/js/vue-support/command.js#L68-L77) can resolve the issue by selecting commands with `CMD_NO_OP` substring or selecting the first item in the `_datastore.commands` as the default selected cmd.

Replace:

```javascript
let selected = _datastore.commands["cmdDisp.CMD_NO_OP"];
```

with:

```javascript
let keys = Object.keys(_datastore.commands).filter(
    command_name => {
        return command_name.indexOf("CMD_NO_OP") != -1;
    });
let selected = (keys.length != 0) ? _datastore.commands[keys[0]] : Object.keys(_datastore.commands)[0];
```
*Note: The fix above was used on an internal project and is being proposed here to be used in the next F Prime release.*

**Comment by saba-ja** [comment]
Fixed in PR #476
